<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="lst_playlists">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>119</y>
      <width>250</width>
      <height>336</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fantasque Sans Mono</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#1DB954</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_uri">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>291</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fantasque Sans Mono</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#1DB954</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_search">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>88</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fantasque Sans Mono</family>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#1DB954</string>
    </property>
    <property name="text">
     <string>Check</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_chro">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>93</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fantasque Sans Mono</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#1DB954</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_track_img">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>240</width>
      <height>240</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fantasque Sans Mono</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#1DB954</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_create_xml">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fantasque Sans Mono</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#1DB954</string>
    </property>
    <property name="text">
     <string>UPDATE_XML</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_track_artist">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fantasque Sans Mono</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#1DB954</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_track_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>415</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fantasque Sans Mono</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#1DB954</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_show_track">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fantasque Sans Mono</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#1DB954</string>
    </property>
    <property name="text">
     <string>Show Track Name And Artist</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_info">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fantasque Sans Mono</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#1DB954</string>
    </property>
    <property name="text">
     <string>INFO</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
